content/articles/an-introduction-to-blockchain-architecture/index.md

```diff
@@ -0,0 +1,113 @@
+### An Introduction to Blockchain Architecture
+As a financial market analyst, I recently ventured into the cryptocurrency asset class. I was astonished by the price delivery mechanism of cryptocurrencies and the huge number of assets commonly referred to as tokens in the cryptocurrency market.
```

Comment:
Can you explain the price delivery mechanism briefly and attach a link to the article discussing the same in-depth.

Reply:
By price delivery mechanism, I meant it’s efficiency in delivering price for effective trading with little volatility as compared to the days when it was early in its adoption phase.